shared/routes/Repl/index.html

```diff
@@ -106,9 +107,23 @@
 		display: none;
 	}
 
+	h2 {
+		margin: 1em 0 0 0;
+		padding: 0 8px;
+		font-weight: 500;
+		font-size: 1.2em;
+	}
+
 	@media (min-width: 768px) {
+		.show-if-mobile {
+			display: none;
+		}
+
 		.repl-container {
+			min-height: auto;
 			height: calc(100vh - 4em);
+			background-color: white;
```

Comment:
There's no benefit to using a hex value over something like `white`. Hex values make sense when you need to define a very specific colour that doesn't have a named equivalent. All the CSS gets minified, so it really doesn't make a difference which format is used in a given place. I prefer using named colours where possible because it's more readable.

The `color` value will be inherited from the body. Setting it again is pointless — it's an unnecessary extra declaration that adds technical debt (if the body colour changes, we'd need to remember to update this one too otherwise they'd get out of sync).